### PR TITLE
test: branch for fix of ci

### PR DESCRIPTION
**Pull request by marbar3778**
- testing replacing syscall with os.FindProces

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: cmd/tendermint/commands/debug/kill.go [hunk 1]
```diff
@@ -121,7 +121,12 @@ func killProc(pid uint64, dir string) error {
 	go func() {
 		// Killing the Tendermint process with the '-ABRT|-6' signal will result in
 		// a goroutine stacktrace.
-		if err := syscall.Kill(int(pid), syscall.SIGABRT); err != nil {
+		p, err := os.FindProcess(os.Getpid())
+		if err != nil {
+			fmt.Printf("failed to find PID to kill Tendermint process: %s", err)
+		}
+		if err = p.Signal(syscall.SIGABRT); err != nil {
```
Comment (alessio) [inline cmd/tendermint/commands/debug/kill.go]: If err is not nil here, p is likely nil

File: cmd/tendermint/commands/debug/kill.go [hunk 2]
```diff
@@ -121,7 +121,12 @@ func killProc(pid uint64, dir string) error {
 	go func() {
 		// Killing the Tendermint process with the '-ABRT|-6' signal will result in
 		// a goroutine stacktrace.
-		if err := syscall.Kill(int(pid), syscall.SIGABRT); err != nil {
+		p, err := os.FindProcess(os.Getpid())
+		if err != nil {
+			fmt.Printf("failed to find PID to kill Tendermint process: %s", err)
```
Comment (melekes) [inline cmd/tendermint/commands/debug/kill.go]:
```suggestion
			fmt.Fprintf(os.Stderr, "failed to find PID to kill Tendermint process: %s", err)
```

File: cmd/tendermint/commands/debug/kill.go [hunk 3]
```diff
@@ -121,7 +121,12 @@ func killProc(pid uint64, dir string) error {
...
+		}
+		if err = p.Signal(syscall.SIGABRT); err != nil {
+			// if err := syscall.Kill(int(pid), syscall.SIGABRT); err != nil {
```
Comment (melekes) [inline cmd/tendermint/commands/debug/kill.go]: remove?

File: cmd/tendermint/commands/debug/kill.go (hunk 1 above)
Comment (alessio) [reply cmd/tendermint/commands/debug/kill.go]: Any comment on this?

Comment (melekes) [reply cmd/tendermint/commands/debug/kill.go]: Oops. Looks like we need an `else` branch. Thanks @alessio!

Comment (melekes) [reply cmd/tendermint/commands/debug/kill.go]: https://github.com/tendermint/tendermint/pull/4271

**Pull request merged**